File: selenium-tests/javascript/selenium_test.js
const { Builder, By, until } = require('selenium-webdriver');

(async function addToCart() {
    let driver = await new Builder().forBrowser('chrome').build();
    try {
        await driver.get('https://www.saucedemo.com/');
        
        await driver.findElement(By.id('user-name')).sendKeys('standard_user');
        await driver.findElement(By.id('password')).sendKeys('secret_sauce');
        await driver.findElement(By.id('login-button')).click();

        await driver.wait(until.elementLocated(By.className('inventory_item')), 5000);
        await driver.findElement(By.css('.btn_inventory')).click();
        
        let cartCount = await driver.findElement(By.className('shopping_cart_badge')).getText();
        console.log(`Productos en el carrito: ${cartCount}`);

    } finally {
        await driver.quit();
    }
})();

File: selenium-tests/python/selenium_test.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.common.keys import Keys
import time

driver = webdriver.Chrome()
driver.get("https://www.google.com")

search_box = driver.find_element(By.NAME, "q")
search_box.send_keys("Selenium WebDriver")
search_box.send_keys(Keys.RETURN)

time.sleep(2)
results = driver.find_elements(By.CSS_SELECTOR, "h3")
for result in results[:5]:
    print(result.text)

driver.quit()
